packages/scss/src/components/verticalNavigation/index.scss

```diff
@@ -3,6 +3,10 @@
 .verticalNavigation {
 	@include vars;
 	@include component;
+
+	&.mod-iconless {
```

Comment:
```suggestion
	&.mod-iconLess {
```

we are not in camelCase here?

Reply:
It's used as a suffix imho 🙂

Reply:
`.mod-noIcon` might be easier to understand?

Reply:
Agree, but we already have iconless references in a couple of other components so it would break consistency.